src/richie/plugins/large_banner/choices.py

```diff
@@ -0,0 +1,18 @@
+"""
+Large banner plugin field choices
+"""
+from django.conf import settings
+
+
+def get_template_choices():
+    """
+    Return list of template choices.
+    """
+    return settings.RICHIE_LARGEBANNER_TEMPLATES
```

Comment:
I don't understand why you make these 2 functions.
I propose to follow the same pattern as here: https://github.com/openfun/richie/blob/master/src/richie/apps/courses/defaults.py

Which means:
- rename the file to defaults.py
- replace these functions by:

```python
    TEMPLATES = return getattr(settings, "RICHIE_LARGEBANNER_TEMPLATES", (
        "richie/large_banner/large_banner.html", _("Default")
    ) 
```

Use this variable in your model with:
```python
choices=TEMPLATES
default=TEMPLATES[0][0],
```